src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,5 @@
+export enum ErrorMessage {
+  ADD = 'Unable to add a todo',
+  DELETE = 'Unable to delete a todo',
+  UPDATE = 'Unable to update a todo',
```

Comment:
```suggestion
export enum ErrorMessage {
  Add = 'Unable to add a todo',
  Delete = 'Unable to delete a todo',
  Update = 'Unable to update a todo',
```
Enum properties should be in PascalCase, not upper case